tests/test_cli/write_cfg_sbcmd_input.txt

```diff
@@ -27,3 +27,39 @@ False
 
 
 
+
```

Comment:
Question: why does this output so many blank lines?

Reply:
`cfg write` takes the default value of an entry in cfg in case the end user enters a newline when asked what they want the value of that entry to be.

The `cfg write` only modifies a single parameter, for brevity, and so all the others can simply be newlines.

This file is what is fed into the test to simulate the end user pressing enter on their keyboards to take the default value of an entry.

Reply:
Can you modify this so that when it fails it gives an informative error? Currently when a new option is added the tests fail with something line "EOF detected" which leaves you with no choice but to debug the tests manually.

Reply:
> Can you modify this so that when it fails it gives an informative error?
@eulertour 
I've made it remind the tester to modify the input file if they haven't. Will that be enough?